pycbc/events/stat.py

```diff
@@ -28,33 +28,6 @@
 import numpy
 from . import events
 
-def get_statistic(option, files):
```

Comment:
Tom, 

What I'm saying is simply to redefine this function to be what you've added to pycbc_coinc_findtrigs. This leaves the signature the same, and also avoids code duplication down the line. (i.e. the entire try except section)